osu.Game/Screens/Select/SongSelect.cs

```diff
@@ -162,7 +162,7 @@ private void load(AudioManager audio, OsuColour colours, ManageCollectionsDialog
                 BleedBottom = Footer.HEIGHT,
                 SelectionChanged = updateSelectedBeatmap,
                 BeatmapSetsChanged = carouselBeatmapsLoaded,
-                FilterApplied = updateVisibleBeatmapCount,
+                FilterApplied = () => Scheduler.Add(updateVisibleBeatmapCount),
```

Comment:
Why does this `Scheduler.Add()` while `carouselBeatmapsLoaded()` does `Scheduler.AddOnce()`?

Reply:
Just an oversight